websites/G/GameBanana/presence.ts

```diff
@@ -0,0 +1,383 @@
+const presence = new Presence({ clientId: "958520351158050887" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let header: HTMLElement;
```

Comment:
The value of `header` is never changed, you should use a constant instead. 
The type `HTMLElement` should not be used in this case as you can assign the proper type of the element instead